intake_thredds/source.py

```diff
@@ -69,6 +69,8 @@ def __init__(
         self.urlpath = url
         if 'simplecache::' in url:
             self.metadata.update({'fsspec_pre_url': 'simplecache::'})
+        assert isinstance(path, list), 'path must be list of str'
+        assert isinstance(path[0], str), 'path must be list of str'
```

Comment:
alternatively we could convert a given str into a list
```suggestion
        if isinstance(path, str):
            path = list(path)
        assert isinstance(path, list), 'path must be list of str'
        assert isinstance(path[0], str), 'path must be list of str'
```

Reply:
`path = list(path)` -> `path = [path]`